src/components/text/Text.driver.new.ts

```diff
@@ -5,7 +5,7 @@ import {usePressableDriver} from '../../testkit/new/usePressable.driver';
 export const TextDriver = (props: ComponentProps) => {
   const driver = usePressableDriver<TextProps>(useComponentDriver(props));
 
-  const getText = () => {
+  const getText = (): React.ReactNode | undefined => {
```

Comment:
Doesn't it suppose to return a string?

Reply:
I thought so, and then I saw it does not, although I think we do compare it to string in tests, maybe we should return `as string | undefined` so it's more clear?
I am really not sure

Reply:
As you've mentioned, this can technically be a `Text` child